src/content/contribute/writing-a-loader.md

```diff
@@ -6,6 +6,7 @@ contributors:
   - michael-ciniawsky
   - byzyk
   - anikethsaha
+  - jamesgeorge007
```

Comment:
Why this change?

Reply:
In regard to https://github.com/webpack/webpack.js.org/pull/3493. The list wasn't updated as part of that PR :sweat_smile: